app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      def index
+        @rents = Rent.where(user_id: params[:user_id])
+        render_paginated @rents
+      end
+
+      def create
+        @rent = Rent.new(rent_params)
+        if @rent.save
```

Comment:
Thougth this is tottally fine, according to the "ruby gurus" a good practice of doing if's is something like this:
```
def method
  return something_bad if there_was_a_problem
  somethin_good
end
```
However, you can do whatever you want with the if/else but i dont care about that specifical practice. It makes the if/else have less lines, but the way you did it is easy to see and understand. Maybe you see that kind of clauses in diferents proyects